scenarioo-client/app/styles/scenarioo.less

```diff
@@ -78,6 +78,14 @@ h4 {
   width: 150px !important;
 }
 
+.btn-default.focus, .btn-default:focus {
```

Comment:
Is a global override wanted here?

Reply:
yes, I think it makes sense. Having same background color for focused buttons as if they were toggled makes no sense to me. Still the focused buttons are identifieable, because it gets a different border color